package.json

```diff
@@ -44,13 +44,23 @@
     "gulp-rename": "1.2.2",
     "gulp-replace": "0.5.4",
     "gulp-zip": "3.1.0",
+    "jasmine-core": "2.4.1",
     "jsdoc": "3.4.0",
     "jshint": "2.9.1",
     "jshint-stylish": "2.1.0",
+    "karma": "0.13.15",
```

Comment:
A lot of these node modules are really out of date. To fix this, run `npm install -g npm-check-updates` on your machine and then run `ncu` inside of the building directory. It will inform you of all packages that need updating. Then you can run `ncu -u` to actually do the update. Then it's just a matter of npm install and testing that things still work.

Do not update `strip-comments` or `gulp-zip`, since both of those have outstanding issues with newer version.  You'll want to merge master into this branch first since there are already some packages that have been updated.